File: C++/Max Max in an array.cpp
#include <iostream>
using namespace std;

int getMax(int arr[], int n)
{

  int max = INT_MIN;
  for (int i = 0; i < n; i++)
  {

    if (arr[i] >= max)
    {
      max = arr[i];
    }
  }

  return max;
}
int getMin(int arr[], int n)
{

  int min = INT_MAX;
  for (int i = 0; i < n; i++)
  {

    if (arr[i] <= min)
    {
      min = arr[i];
    }
  }

  return min;
}

int main()
{

  int size;
  cout << "Enter the array size: " << endl;
  cin >> size;

  int num[100];

  cout << "Enter array elements: " << endl;

  // Taking input in array
  for (int i = 0; i < size; i++)
  {
    cin >> num[i];
  };

  int maximum = getMax(num, size);
  int minimum = getMin(num, size);
  cout << "Max: " << maximum << " " << "Min" << minimum << endl;

  return 0;
}

File: Python/Max Max in an array.py
def getMax(arr):
    max_val = float("-inf")
    for i in range(len(arr)):
        if arr[i] >= max_val:
            max_val = arr[i]
    return max_val


def getMin(arr):
    min_val = float("inf")
    for i in range(len(arr)):
        if arr[i] <= min_val:
            min_val = arr[i]
    return min_val


# Taking user input
size = int(input("Enter the array size: "))
print("Enter array elements:")

num = []
for i in range(size):
    num.append(int(input()))

maximum = getMax(num)
minimum = getMin(num)
print(f"Max: {maximum} Min: {minimum}")
